containers/podman-desktop
# feat: enable lightmode on multiple form pages

### What does this PR do?

This PR adds light mode to almost all form pages. There are pages that are not here bc they are a bit more complex and it would be easier to review them separately. All you find here required basic changes and most of them have similar design.

### Screenshot / video of UI

i won't include any screenshot as they are a lot, it's more convenient you check them manually by running desktop.

### What issues does this PR fix or reference?

it is part of #7214 

### How to test this PR?

The pages to verify:

1. `ExportContainer`: go to containers page -> click on the "three vertical dots" icon of a container -> export container
2. `ImportImage`: go to the images page -> click on import (top right)
3. `LoadImages`: go to the images page -> click on load(top right)
4. `PullImage`: go to the images page -> click on pull(top right)
5. `RunImage`:  go to the images page -> click on the start action of an image
6. `SaveImages`: go to images page -> click on the "three vertical dots" icon of an image-> save image
7. `DeployToKube`: go to the pods page -> click on the "three vertical dots" icon of an image-> deploy to kube
8. `PodCreateFromContainers`: go to containers page -> select 2 or more containers -> click on `create pod` on the top
9. `CreateVolume`: go to volumes page -> click on create (top right)
